broadcasting.md

```diff
@@ -247,6 +255,47 @@ npm run dev
 > **Note**  
 > To learn more about compiling your application's JavaScript assets, please consult the documentation on [Vite](/docs/{{version}}/vite).
 
+Additional supported features :
+
+*1. Modify private/presence channel capability. Default: Full capability*
+- Channel access control rights are granted for each individual user separately using `ably-capability`.
```

Comment:
```suggestion
- Channel access control rights are granted for each individual user separately using an `ably-capability` object.
```

Reply:
changed to `using an ably-capability array`